src/app/app.module.ts

```diff
@@ -0,0 +1,35 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { AppRoutingModule } from './app-routing.module';
+import { MaterialModuleModule } from './common/material-module/material-module.module';
+import { AppComponent } from './app.component';
+import { PageNotFoundComponent } from './invalid/page-not-found/page-not-found.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { NoauthComponent } from './layouts/noauth/noauth.component';
+import { HeaderComponent } from './layouts/noauth/header/header.component';
+import { FooterComponent } from './layouts/noauth/footer/footer.component';
+import { ReactiveFormsModule } from '@angular/forms';
+import { DashboardComponent } from './main/dashboard/dashboard.component';
+import { APP_BASE_HREF, HashLocationStrategy, LocationStrategy } from '@angular/common';
+@NgModule({
+  declarations: [
+    AppComponent,
+    PageNotFoundComponent,
+    NoauthComponent,
+    HeaderComponent,
+    FooterComponent,
+    DashboardComponent,
+
+  ],
+  providers: [{ provide: APP_BASE_HREF, useValue: '/' },
```

Comment:
APP BASE HREF is used for Path location based strategy. If you decide to use HashLocationBased strategy , no need to use it.